staging/src/k8s.io/apiextensions-apiserver/pkg/apis/apiextensions/types.go

```diff
@@ -267,7 +267,7 @@ const (
 
 // CustomResourceDefinitionCondition contains details for the current condition of this pod.
 type CustomResourceDefinitionCondition struct {
-	// Type is the type of the condition.
+	// Type is the type of the condition. Can be Established, NamesAccepted, or Terminating.
```

Comment:
Leave it more open-ended than this, since other conditions can be recorded. For example: "Types include …"

Reply:
Agree. Updated the description, PTAL.